phpcs.xml.dist

```diff
@@ -23,6 +23,6 @@
 	<exclude-pattern>*/ui/lib/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 	<exclude-pattern>*/build/*</exclude-pattern>
-	<exclude-pattern>*/local/public/*</exclude-pattern>
+	<exclude-pattern>*/local/*</exclude-pattern>
```

Comment:
Since it only used is in development, I've added the whole `local` directory to be ignored by PHPCS.

Reply:
@kasparsd @derekherman :point_up:

Reply:
It would be nice to have all PHP files with consistent formatting even if they're used for development purposes.